package.json

```diff
@@ -57,113 +57,37 @@
     ]
   },
   "devDependencies": {
-    "@babel/register": "^7.4.4",
-    "@babel/standalone": "^7.3.2",
-    "@microsoft/api-extractor": "^7.5.2",
-    "@mdx-js/loader": "^1.0.21",
-    "@types/babel__traverse": "^7.0.4",
-    "@types/color": "^3.0.0",
     "@types/enzyme": "^3.9.3",
-    "@types/express": "^4.16.1",
-    "@types/faker": "^4.1.3",
     "@types/gulp": "^4.0.6",
-    "@types/gulp-babel": "6.1.29",
-    "@types/gulp-cache": "^0.4.4",
-    "@types/gulp-load-plugins": "^0.0.31",
-    "@types/gulp-remember": "^0.0.31",
     "@types/lodash": "^4.14.118",
     "@types/node": "^10.3.2",
     "@types/prop-types": "^15.7.1",
-    "@types/puppeteer": "^1.11.1",
     "@types/react": "^16.8.10",
-    "@types/react-custom-scrollbars": "^4.0.5",
     "@types/react-dom": "^16.8.3",
-    "@types/react-is": "^16.7.1",
-    "@types/react-router-dom": "^4.3.4",
-    "@types/rimraf": "^2.0.2",
-    "@types/webpack-dev-middleware": "^2.0.1",
-    "@types/webpack-hot-middleware": "^2.16.3",
-    "babel-loader": "^8.0.6",
-    "circular-dependency-plugin": "^5.0.2",
-    "clean-webpack-plugin": "^0.1.19",
-    "color": "^3.1.0",
-    "connect-history-api-fallback": "^1.3.0",
-    "copy-to-clipboard": "^3.0.8",
-    "copy-webpack-plugin": "^4.5.2",
     "cross-env": "^5.1.4",
     "danger": "^6.0.5",
-    "del": "^5.1.0",
     "doctoc": "^1.3.0",
-    "doctrine": "^2.0.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
-    "express": "^4.15.4",
-    "extract-comments": "^1.0.0",
-    "faker": "^4.1.0",
-    "flat": "^4.1.0",
-    "fork-ts-checker-webpack-plugin": "^1.3.3",
     "gh-pages": "^1.0.0",
-    "glob": "^7.1.2",
     "gulp": "^4.0.2",
-    "gulp-babel": "^8.0.0",
-    "gulp-cache": "^1.0.2",
-    "gulp-debug": "^4.0.0",
-    "gulp-load-plugins": "^1.5.0",
-    "gulp-plumber": "^1.2.0",
-    "gulp-remember": "^1.0.1",
-    "gulp-transform": "^3.0.5",
-    "gulp-typescript": "^5.0.0",
-    "gulp-util": "^3.0.8",
-    "html-webpack-plugin": "^4.0.0-beta.5",
-    "husky": "^0.14.3",
-    "json-stable-stringify-without-jsonify": "^1.0.1",
     "lerna": "^3.10.7",
     "lerna-alias": "^3.0.3-0",
-    "license-webpack-plugin": "^2.1.1",
     "lint-staged": "^7.0.2",
-    "markdown-table": "^1.1.3",
-    "merge2": "^1.2.2",
-    "normalize.css": "^8.0.0",
-    "portfinder": "^1.0.20",
     "prettier": "^1.19.1",
-    "progress": "^2.0.3",
-    "puppeteer": "^1.11.0",
     "react": "^16.8.6",
-    "react-ace": "^5.1.2",
-    "react-codesandboxer": "^3.1.3",
-    "react-custom-scrollbars": "^4.2.1",
-    "react-document-title": "^2.0.3",
     "react-dom": "^16.8.6",
```

Comment:
`react` seems also should be deleted

Reply:
`react` and a few other things need to be moved to dependencies of a lot of individual packages, so I think I'm also going to tackle that one separately.